doc/luasnip.txt

```diff
@@ -1569,6 +1569,23 @@ there are multiple) the associated file to edit.
     editing. The default is a simple `vim.cmd("edit " .. file)` (replace the
     current buffer), but one could open the file in a split, a tab, or a floating
     window, for example.
+- `extend`: `fn(ft:string, default_paths:table)` -> `table`
```

Comment:
default_paths: `table` -> `string[]`?
And `(string,string)[]` for the return value?

Also, I think mentioning how exactly the return values will be used would be nice
(first -> item-name, second -> item-path basically, more words ofc :D)